src/app/loadout-builder/hooks/useProcess.ts

```diff
@@ -75,30 +76,41 @@ export function useProcess(
 
     const processItems: ProcessItemsByBucket = {};
     const itemsById: { [id: string]: D2Item } = {};
+    const classItemsById: { [id: string]: D2Item[] } = {};
+    const classItems: ProcessItem[] = [];
 
     for (const [key, items] of Object.entries(filteredItems)) {
       processItems[key] = [];
       for (const item of items) {
         if (item.isDestiny2()) {
-          processItems[key].push(
-            mapDimItemToProcessItem(item, lockedArmor2ModMap[bucketsToCategories[item.bucket.hash]])
+          const processItem = mapDimItemToProcessItem(
+            item,
+            lockedArmor2ModMap[bucketsToCategories[item.bucket.hash]]
           );
-          itemsById[item.id] = item;
+          if (key !== LockableBuckets.classitem.toString()) {
```

Comment:
You can probably extract `LockableBuckets.classitem.toString()` from the for loop

Reply:
Removed now